util/build_compilation_database.sh

```diff
@@ -12,8 +12,8 @@ MY_CMAKE_FLAGS=("-GNinja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_EXPORT_COMPILE_COM
 
 mkdir -p "${AWS_LC_BUILD}"
 
-cmake "${BASE_DIR}" -B "${AWS_LC_BUILD}" ${MY_CMAKE_FLAGS[@]} "${@}"
+cmake "${BASE_DIR}" -B "${AWS_LC_BUILD}" "${MY_CMAKE_FLAGS[@]}" "${@}"
 
-cmake --build "${AWS_LC_BUILD}" --target all
+cmake --build "${AWS_LC_BUILD}" -j 4 --target all_tests
```

Comment:
nit: why 4 here? should we use `$(nproc)` or [the integration tests' logic](https://github.com/aws/aws-lc/blob/1fbf58437a32da4008774eed92530b787d210b1e/tests/ci/common_posix_setup.sh#L24)?

Reply:
B/c it provides an easy way to perform the build, I was running the script on Windows.  It's not really relevant to this PR otherwise.  It seemed to be defaulting to 0 with "Visual Studio" generator, so I changed it to 4 which provides a reasonable level of parallelism for most hosts.